ui/src/ethereum/walletConnect.ts

```diff
@@ -140,7 +154,28 @@ export class WalletConnectClient implements WalletConnect {
   // and
   // https://github.com/WalletConnect/walletconnect-monorepo/pull/370#issuecomment-776038638
   private reinit() {
-    this.connector = createConnector();
+    this.connector = new Connector({
+      bridge: "https://radicle.bridge.walletconnect.org",
```

Comment:
I thought that we should be using their default bridge now?

Reply:
This was a merge conflict. Fixed in https://github.com/radicle-dev/radicle-upstream/pull/2276/commits/a9becf75c0436f9718d51e6dc8df03c5b33bc96f